.github/workflows/remix_test.yml

```
@@ -0,0 +1,25 @@
+name: Test Remix
+
+on:
+  pull_request:
+    paths:
+      - remix/**
+
+jobs:
+  test-remix:
+    name: Run tests
+    runs-on: ubuntu-latest
```

To avoid repetition in defining `working-directory,` we should be able to do:

```suggestion
    runs-on: ubuntu-latest
    defaults:
    	working-directory: remix
```

See https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-setting-default-run-step-options-for-a-job